compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -648,9 +648,11 @@ class OMR_EXTENSIBLE CodeGenerator
 
    // high precision timer can be converted into millisecond timer or nanosecond timer through appropriate
    // division or multiplication (respectively). For example, 390 high precision timer provides accuracy to
-   // 1/8 of a microsecond so converters are 8000 and 125 respectively (multiplying the two should give you 1000000)
+   // 1/2048 of a microsecond so, to convert maximum-precision timer to
+   // - milliseconds, divide maximum-precision timer by 2,048,000
+   // - nanoseconds, multiply maximum-precision timer by 125/256
    int64_t getCurrentTimeMillisDivisor() { return 0; } // no virt
-   int64_t getNanoTimeMultiplier() { return 0; } // no virt
```

Comment:
Sure, but keep in mind that after moving these constants into the upstream projects, OMR will still have mentions of these constants in port library. We already don't have one single method for getting e.g. `CurrentTimeMillisDivisor`, now we are gonna put this constant into two separate projects.

I'm worried about a scenario in which some updates the constant in OMR without updating it in the upstream project. This can lead to nasty bugs.